torch/nn/functional.py

```diff
@@ -3829,26 +3829,125 @@ def fold(input, output_size, kernel_size, dilation=1, padding=0, stride=1):
 def _pad_circular(input, padding):
     # type: (Tensor, List[int]) -> Tensor
     """
-    Arguments
-        :param input: tensor of shape :math:`(N, C_{\text{in}}, H, [W, D]))`
-        :param padding: (tuple): m-elem tuple where m is the degree of convolution
-    Returns
-        :return: tensor of shape :math:`(N, C_{\text{in}}, [D + 2 * padding[0],
-                 H + 2 * padding[1]], W + 2 * padding[2]))`
+    Args:
+        input: Tensor that follows the formatting of the input to convolution
```

Comment:
The original documentation was more precise about this, and this should also be written to not require a second lookup. Also, this expression is ambiguous because different convolutions accepts different input tensors. 

Our convolution documentation is very poor (as is convolution documentation generally), so we should strive to be extremely clear here about what this function does and how to use it.

Reply:
I improved the documentation. The description for input is similar to what it was before. I included examples to help illustrate how the input parameters work.